iina/MPVController.swift

```diff
@@ -517,8 +517,9 @@ class MPVController: NSObject {
 
     // As mpv support for audio using the AVFoundation framework is new we enable it before applying
     // user's settings. This allows a user to roll back to the Core Audio framework should a problem
```

Comment:
This comment was about the user being able to add "ao=coreaudio" in advanced settings to rollback from AVFoundation to Core Audio. So the comment is a bit out of date now that we have an explicit setting for this.